lib/engine/game/g_1822_ca/game.rb

```diff
@@ -63,9 +63,10 @@ class Game < G1822::Game
 
         GAME_END_REASONS_TIMING_TEXT = {
           current_or: 'Next end of an OR',
-          full_or: 'Next end of a complete OR set, with one additional OR added on',
+          full_or: 'Next end of a complete OR set, with one additional OR added on ' \
+                   '(if bank breaks during an OR). Next end of an OR (if bank breaks during an SR).',
         }.freeze
-
+                
```

Comment:
There was another rubocop failure for trailing whitespace here https://github.com/tobymao/18xx/actions/runs/19454064271/job/55668501481?pr=12254

> Offenses:
>
> lib/engine/game/g_1822_ca/game.rb:69:1: C: [Correctable] Layout/TrailingWhitespace: Trailing whitespace detected.

```suggestion

```